README_OLD.md

```diff
@@ -0,0 +1,320 @@
+# Stackdriver Error Reporting for Node.js
```

Comment:
This file should go away,  yes?

Reply:
Yes, I should merge `README_OLD.md` into `README.md`, but I think that can be done on a follow-up PR.